### PR TITLE
ci: Fix prettier auto-formatting (no-changelog)

### PR DESCRIPTION
**Pull request by netroy**

### CODE REVIEW

**Comment by ivov** [comment]
@netroy What issue is this fixing? Autoformatting appears to be working on my end.

**Comment by netroy** [comment]
> @netroy What issue is this fixing? Autoformatting appears to be working on my end.

It's not working on my vscode setup. Do you have a global installation of prettier?

**Comment by ivov** [comment]
> > @netroy What issue is this fixing? Autoformatting appears to be working on my end.
> 
> It's not working on my vscode setup. Do you have a global installation of prettier?

No globally installed Prettier:

```
npm list -g --depth=0
/opt/homebrew/lib
├── @vscode/vsce@2.19.0
├── depcheck@1.4.3
├── dpdm@3.12.0
├── iterm2-tab-set@0.7.1
├── n8n-node-dev@0.100.0
├── node-gyp@9.3.1
├── npm-check-updates@16.10.12
├── npm@9.8.0
├── pnpm@8.6.2
└── ts-node@10.9.1
```

And `settings.json` is identical to `settings.default.json` except for unrelated color customizations.

No need to spend long on this - I'll test this out and if it makes no difference on my end I'll just approve.

File: packages/editor-ui/index.html [hunk 1]
```diff
@@ -9,7 +9,49 @@
 			window.BASE_PATH = '/{{BASE_PATH}}/';
 			window.REST_ENDPOINT = '{{REST_ENDPOINT}}';
 		</script>
-		<script>!function(t,e){var o,n,p,r;e.__SV||(window.posthog=e,e._i=[],e.init=function(i,s,a){function g(t,e){var o=e.split(".");2==o.length&&(t=t[o[0]],e=o[1]),t[e]=function(){t.push([e].concat(Array.prototype.slice.call(arguments,0)))}}(p=t.createElement("script")).type="text/javascript",p.async=!0,p.src=s.api_host+"/static/array.js",(r=t.getElementsByTagName("script")[0]).parentNode.insertBefore(p,r);var u=e;for(void 0!==a?u=e[a]=[]:a="posthog",u.people=u.people||[],u.toString=function(t){var e="posthog";return"posthog"!==a&&(e+="."+a),t||(e+=" (stub)"),e},u.people.toString=function(){return u.toString(1)+".people (stub)"},o="capture identify alias people.set people.set_once set_config register register_once unregister opt_out_capturing has_opted_out_capturing opt_in_capturing reset isFeatureEnabled getFeatureFlag onFeatureFlags reloadFeatureFlags".split(" "),n=0;n<o.length;n++)g(u,o[n]);e._i.push([i,s,a])},e.__SV=1)}(document,window.posthog||[])</script>
+		<script>
+			!(function (t, e) {
+				var o, n, p, r;
+				e.__SV ||
```
Comment (ivov) [inline packages/editor-ui/index.html:15]: Does it make sense to format this?

**Review by ivov — DISMISSED** [review]
Still working on my end.

File: packages/editor-ui/index.html (hunk 1 above)
Comment (netroy) [reply packages/editor-ui/index.html:15]: I just ran `pnpm format`. Should we put this in the ignore file ?

Comment (ivov) [reply packages/editor-ui/index.html:15]: I'd add it but super minor thing, up to you

Comment (netroy) [reply packages/editor-ui/index.html]: done

**Review by ivov — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.6.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.6.0)

**Pull request merged**